freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -107,6 +111,19 @@ public SyncOperationStatus getSyncOperationStatus(@NotNull String operationId) {
                 operationService.getOperationForAccountIdAndOperationId(accountId, operationId));
     }
 
+    @Override
+    public String getKeytab(@NotNull String environmentCrn, String targetUserCrn) {
+        String callerUserCrn = checkUserCrn();
+        LOGGER.debug("getKeytab() request for environmentCrn={}, callerUserCrn={}, targetUserCrn={}", environmentCrn, callerUserCrn, targetUserCrn);
+        // TODO For now, we ignore the optional targetUserCrn and only allow retrieving keytab for the calling user. When
+        //      we later enable this, we need to make sure that the calling user has the necessary authorization / right to get
+        //      keytab for another user.
+        if (targetUserCrn != null && !callerUserCrn.equals(targetUserCrn)) {
+            throw new BadRequestException("Retrieving a keytab for another user is not yet supported");
+        }
+        return userKeytabService.getKeytabBase64(callerUserCrn, environmentCrn);
```

Comment:
call this with the targetUserCrn rather than the actorCrn. The actorCrn will be available to other code in through the ThreadBasedUserCrnProvider (I think I remembered that correctly)